index.html

```diff
@@ -140,10 +140,15 @@
           "description": "A convenience constructor for RDF quads.",
           "implements": [ "DataFactory" ]
         }, {
-          "@id": "https://github.com/rubensworks/rdf-string.js  ",
+          "@id": "https://github.com/rubensworks/rdf-string.js",
           "name": "RDF String",
           "description": "Convenience functions for creating and serializing RDF terms and quads.",
           "implements": [ "DataFactory" ]
+        }, {
+          "@id": "https://github.com/linkeddata/rdflib.js",
+          "name": "rdflib.js",
+          "description": "Javascript RDF library for browsers and Node.js.",
+          "implements": []
```

Comment:
It looks like it does implement DataFactory interface, would you like to add it here?
https://github.com/linkeddata/rdflib.js/blob/master/src/data-factory.js#L58

Reply:
@ktk :point_up:

Reply:
ok if it indeed does implement it I'm ok with it